src/amo/components/InstallButtonWrapper/index.js

```diff
@@ -46,6 +47,7 @@ type InternalProps = {|
   i18n: I18nType,
   installStatus: $PropertyType<InstalledAddon, 'status'>,
   userAgentInfo: UserAgentInfoType,
+  showButtonInsteadOfLink: boolean,
```

Comment:
As above, this only needs to be in `Props`. Also see my comment about the prop name, and the fact that it should default to `false`.